gazel/gazel.go

```diff
@@ -642,6 +648,8 @@ func ReconcileRules(pkgPath string, rules []*bzl.Rule, dryRun bool) (bool, error
 		reconcileAttr(o, r, "srcs")
 		reconcileAttr(o, r, "deps")
 		reconcileAttr(o, r, "library")
+		reconcileAttr(o, r, "openapi_targets")
```

Comment:
Even if this is not enabled, we will still muck with these. That's probably not what we want.